tictectoe-domain/src/main/java/com/example/tictectoe_domain/TictectoeBoard.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.tictectoe_domain
+
+class TictectoeBoard {
+    private lateinit var board: MutableList<Player>
+    private var player = Player.PLAYER1
```

Comment:
```suggestion
    var player = Player.PLAYER1
        private set
```

별도의 get 함수를 만들어주지 않아도 코틀린의 필드는 프로퍼티의 성격을 가지고 있기 때문에 get은 public하게, set은 private하게 하여 getter를 표현할 수 있습니다.